api/src/shared/domain/models/AssessmentResult.js

```diff
@@ -71,92 +67,17 @@ class AssessmentResult {
     this.juryId = juryId;
   }
 
-  static buildAlgoErrorResult({ error, assessmentId, juryId, emitter }) {
-    return new AssessmentResult({
-      emitter,
-      commentByJury: error.message,
-      pixScore: 0,
-      reproducibilityRate: 0,
-      status: status.ERROR,
-      assessmentId,
-      juryId,
-    });
-  }
-
-  static buildStandardAssessmentResult({ pixScore, reproducibilityRate, status, assessmentId, juryId, emitter }) {
-    return new AssessmentResult({
-      emitter,
-      pixScore,
-      reproducibilityRate,
-      status,
-      assessmentId,
-      juryId,
-    });
-  }
-
+  /**
+   * @deprecated : usage of this method is highly discutable since it is using an `Assessment.states.STARTED`
+   *  that is not a valid status know from AssessmentResult (see status enumeration in AssessmentResult)
```

Comment:
```suggestion
* that is not a valid status known from AssessmentResult (see status enumeration in AssessmentResult)
```
Selon ce qu'on veut dire

Reply:
Done